### ISSUE TITLE
separate batch creation and fetching into two pipeline stages?

### ISSUE THREAD

**Issue by rahulbot** [mediacloud/story-indexer#100]
It sounds like there are benefits to re-architecting the initial phase to include:
1) an "ingester"?, which starts with an RSS file of URLs and outputs batches for fetching in parallel
2) a more limited "fetcher" which starts with those batches and outputs full HTML after fetching it

Next step is to propose a technical architecture for this.

**Comment by rahulbot** [comment]
Decided to put our eggs in the queued-based fetcher basket, which means we probably won't do much on this. Close once #128 is done.

**Comment by philbudne** [comment]
queue fetcher is active: closing
